tracer/test/Datadog.Trace.Tests/ServerlessTests.cs

```diff
@@ -89,25 +89,67 @@ public void GetSyncIntegrationTypeFromParamCountMoreThanExpected()
         [Fact]
         public void GetAsyncIntegrationTypeFromParamCountZero()
         {
-            Serverless.GetAsyncIntegrationTypeFromParamCount(1).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaNoParamAsync");
+            Serverless.GetAsyncIntegrationTypeFromParamCount(0).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaNoParamAsync");
         }
 
         [Fact]
         public void GetAsyncIntegrationTypeFromParamCountOne()
         {
-            Serverless.GetAsyncIntegrationTypeFromParamCount(2).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaOneParamAsync");
+            Serverless.GetAsyncIntegrationTypeFromParamCount(1).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaOneParamAsync");
         }
 
         [Fact]
         public void GetAsyncIntegrationTypeFromParamCountTwo()
         {
-            Serverless.GetAsyncIntegrationTypeFromParamCount(3).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaTwoParamsAsync");
+            Serverless.GetAsyncIntegrationTypeFromParamCount(2).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaTwoParamsAsync");
         }
 
         [Fact]
         public void GetAsyncIntegrationTypeFromParamCountMoreThanExpected()
         {
             Assert.Throws<ArgumentOutOfRangeException>(() => Serverless.GetAsyncIntegrationTypeFromParamCount(5));
         }
+
+        [Fact]
+        public void GetVoidIntegrationTypeFromParamCountZero()
+        {
+            Serverless.GetVoidIntegrationTypeFromParamCount(0).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaNoParamVoid");
+        }
+
+        [Fact]
+        public void GetVoidIntegrationTypeFromParamCountOne()
+        {
+            Serverless.GetVoidIntegrationTypeFromParamCount(1).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaOneParamVoid");
+        }
+
+        [Fact]
+        public void GetVoidIntegrationTypeFromParamCountTwo()
+        {
+            Serverless.GetVoidIntegrationTypeFromParamCount(2).Should().Be("Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.LambdaTwoParamsVoid");
+        }
+
+        [Fact]
+        public void GetVoidIntegrationTypeFromParamCountMoreThanExpected()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() => Serverless.GetAsyncIntegrationTypeFromParamCount(5));
```

Comment:
Same comment, perhaps we can use 3 as the argument here?

Reply:
Yes, there are actually tests for params out of bounds (previous merged PR) here : https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/Datadog.Trace.Tests/ServerlessTests.cs